src/github.com/getlantern/balancer/balancer.go

```diff
@@ -57,16 +57,16 @@ func New(dialers ...*Dialer) *Balancer {
 	return bal
 }
 
-// DialQOS dials network, addr using one of the currently active configured
-// Dialers. It attempts to use a Dialer whose QOS is higher than targetQOS, but
-// will use the highest QOS Dialer(s) if none meet targetQOS. When multiple
-// Dialers meet the targetQOS, load is distributed amongst them randomly based
-// on their relative Weights.
-//
-// If a Dialer fails to connect, Dial will keep falling back through the
-// remaining Dialers until it either manages to connect, or runs out of dialers
-// in which case it returns an error.
-func (b *Balancer) DialQOS(network, addr string, targetQOS int) (net.Conn, error) {
+// TrustedDialerAndConn creaetes a balancer.Dialer and a network connection for an HTTP connection
+// (as opposed to HTTPS).
+func (b *Balancer) TrustedDialerAndConn() (*Dialer, net.Conn, error) {
+	// At this point we don't actually have the destination URL. For our purposes, however,
+	// the destination URL is not actually necessary and is not even used in the dial function
+	// because dial is really just dialing the proxy server.
+	return b.dialerAndConn("tcp", "http://www.doesnotexist.com:80", 0)
```

Comment:
Was this URL introduced to force using a trusted dialer? The `addr` will be printed in the logs a few times in case of errors, which could be confusing.